src/app/components/details/release_details.ui

```diff
@@ -1,129 +1,83 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <interface>
-  <requires lib="gtk" version="4.0"/>
-  <requires lib="libadwaita" version="1.0"/>
+  <requires lib="gtk" version="4.0" />
+  <requires lib="libadwaita" version="1.0" />
   <template class="ReleaseDetailsWindow" parent="AdwWindow">
-    <property name="can-focus">0</property>
     <property name="modal">1</property>
-    <property name="default-width">250</property>
+    <property name="hide-on-close">1</property>
+    <property name="default-width">360</property>
     <child>
-      <object class="GtkWindowHandle">
+      <object class="GtkBox">
+        <property name="orientation">vertical</property>
         <child>
-          <object class="GtkBox">
-            <property name="orientation">vertical</property>
+          <object class="AdwHeaderBar">
+            <property name="show-end-title-buttons">1</property>
+            <child type="title">
+              <object class="AdwWindowTitle" id="album_artist" />
+            </child>
+            <style>
+              <class name="flat" />
+            </style>
+          </object>
+        </child>
+        <child>
+          <object class="GtkListBox">
+            <property name="margin-start">6</property>
+            <property name="margin-end">6</property>
+            <property name="margin-top">6</property>
+            <property name="margin-bottom">6</property>
+            <property name="valign">start</property>
+            <property name="selection-mode">none</property>
+            <property name="show-separators">1</property>
+            <style>
+              <class name="card" />
```

Comment:
I'm late, but you use .card for everything *but* lists - boxed-list would be better here.